xsd/netex_part_1/part1_tacticalPlanning/netex_servicePattern_version.xsd

```diff
@@ -561,14 +564,24 @@ Rail transport, Roads and Road transport
 		<xsd:sequence>
 			<xsd:element name="TransportMode" type="AllVehicleModesOfTransportEnumeration" default="all" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>MODE of end Point of TRANSFER . Default is all modes. of SCHEDULED STOP POINT Can be derived.</xsd:documentation>
+					<xsd:documentation>MODE of end Point of CONNECTION. Default is all modes,  MODE of SCHEDULED STOP POINT Can be derived.</xsd:documentation>
```

Comment:
Here you change Transfer to Connection, but at line 572 it is again a Transfer.

Reply:
oups... corrected